README.md

```diff
@@ -1,349 +1,152 @@
-            secret:
-              secretName: ceph-client-conf
-          mounts:
-          - name: ceph
-            mountPath: "/etc/ceph"
-            readOnly: true
-```
-
-The following represents an example of the entire Cinder object that can be used
-to trigger the Cinder service deployment, and enable the Cinder backend that
-points to an external Ceph cluster.
 
+Install the OpenStack operators:
 
-```
-apiVersion: cinder.openstack.org/v1beta1
-kind: Cinder
-metadata:
-  name: cinder
-  namespace: openstack
-spec:
-  serviceUser: cinder
-  databaseInstance: openstack
-  databaseUser: cinder
-  cinderAPI:
-    replicas: 1
-    containerImage: quay.io/podified-antelopecentos9/openstack-cinder-api:current-podified
-  cinderScheduler:
-    replicas: 1
-    containerImage: quay.io/podified-antelopecentos9/openstack-cinder-scheduler:current-podified
-  cinderBackup:
-    replicas: 1
-    containerImage: quay.io/podified-antelopecentos9/openstack-cinder-backup:current-podified
-    customServiceConfig: |
-      [DEFAULT]
-      backup_driver = cinder.backup.drivers.ceph.CephBackupDriver
-      backup_ceph_pool = backups
-      backup_ceph_user = admin
-  secret: cinder-secret
-  cinderVolumes:
-    volume1:
-      containerImage: quay.io/podified-antelopecentos9/openstack-cinder-volume:current-podified
-      replicas: 1
-      customServiceConfig: |
-        [DEFAULT]
-        enabled_backends=ceph
-        [ceph]
-        volume_backend_name=ceph
-        volume_driver=cinder.volume.drivers.rbd.RBDDriver
-        rbd_ceph_conf=/etc/ceph/ceph.conf
-        rbd_user=admin
-        rbd_pool=volumes
-        rbd_flatten_volume_from_snapshot=False
-        rbd_secret_uuid=<Ceph_FSID>
-  extraMounts:
-    - name: cephfiles
-      region: r1
-      extraVol:
-      - propagation:
-        - CinderVolume
-        - CinderBackup
-        extraVolType: Ceph
-        volumes:
-        - name: ceph
-          secret:
-            secretName: ceph-client-conf
-        mounts:
-        - name: ceph
-          mountPath: "/etc/ceph"
-          readOnly: true
+```sh
+eval $(crc oc-env)
+cd ..
+make crc_storage openstack_wait
 ```
 
-When the service is up and running, it's possible to interact with the Cinder
-API and create the Ceph `cinder type` backend which is associated with the Ceph
-tier specified in the config file.
+Create the deployment config for the LVM backend and deploy the OpenStack
+control plane:
 
-## Example: configure Cinder with additional networks
-
-The Cinder spec can be used to configure Cinder to have the pods
-being attached to additional networks to e.g. connect to a Ceph
-RBD server on a dedicated storage network.
+```sh
+oc kustomize \
+  https://github.com/openstack-k8s-operators/cinder-operator.git/config/samples/backends/lvm/iscsi?ref=main \
+  > ~/openstack-deployment.yaml
 
-Create a network-attachement-definition which then can be referenced
-from the Cinder CR.
+oc label node --all openstack.org/cinder-lvm=
```

Comment:
I think if we're going to label "--all" nodes then I recommend adding a note that this works because there's only one node because we're using CRC.

Reply:
Done.